src/ape_ethereum/ecosystem.py

```diff
@@ -224,7 +242,7 @@ def str_to_slot(text):
             outputs=[ABIType(type="string")],
         )
         try:
-            name = ContractCall(abi, address)()
+            name = ContractCall(abi, address)(skip_trace=True)
```

Comment:
Hmm, I missed where `skip_trace` kwarg was introduced. In this PR or another one?

Its just an optimization for internal uses? Why do traces happen normally, and can this be opt in behavior instead of opt out?

Reply:
It was added on a separate PR but missed here.

If you are running tests and need traces (such as running with `--gas`), it would attempt to gather trace data from every single proxy-check call we do here, which made debugging trace-related stuff challenging and also made things slow.

It is opt-in in the sense that it only gathers trace data on calls if it needs it (such as with `--gas`), but if the kwarg were reversed, every call in a project's tests would need the kwarg in order for it to show up in gas report.

Reply:
Gotcha, so it's sort of an internal kwarg that ape injects during moments it needs it